### ISSUE TITLE
Loss calculation

### ISSUE THREAD

**Issue by ashispapu** [tacchinotacchi/distil-bilstm#4]
Thanks for sharing the repo and blog. I need some clarity on the loss calculation. As mentioned in the blog loss is calculated as below.
L=(1−α)LH+αLKL

However in the implementation(**trainer.py, get_loss()**) only computes either of them. Is it inline with the theory as in the blog Or am I missing anything?

**Comment by tacchinotacchi** [comment]
No problem. If you go to part of the section "Setting up the training process for knowledge distillation" where I showed the formula L=(1−α)LH+αLKL, I explain that with this data augmentation algorithm we can't trust the original labels, so α is set to one. The LH would be multiplied by 0, so there's no need to compute it.

EDIT: about the code, `get_loss` is supposed to be multipurpose: it calculates cross-entropy when you use the Trainer class to fine-tune BERT, and KL divergence/MSE when training the student model.

**Comment by ashispapu** [comment]
Thank you for the explanation.
